aten/src/ATen/test/vulkan_api_test.cpp

```diff
@@ -79,6 +79,30 @@ TEST(VulkanAPITest, add_scalar_) {
   ASSERT_TRUE(almostEqual(a_cpu, a_vulkan.cpu()));
 }
 
+TEST(VulkanAPITest, mul_scalar) {
+  const auto a_cpu = at::rand({1, 1, 1, 1}, at::device(at::kCPU).dtype(at::kFloat));
```

Comment:
Please try a different size.  There is nothing special about the sizes I picked - I just picked a bunch of prime numbers, but there's nothing special about it.  The only thing is that I wanted to make sure non power of two numbers and odd dimensions work fine.